nevergrad/functions/functionlib.py

```diff
@@ -63,6 +65,43 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class ML_tuning(ExperimentFunction):
```

Comment:
Please avoid `_` in class name
`MachineLearningTuning` or `MLTuning` would be better

Reply:
Done 👍